### PR TITLE
Fix `quiet` argument of `load_all()`

### LINKED ISSUE #213: quiet = TRUE no longer works
```
> pkgload::load_all(quiet = TRUE)
ℹ Loading pkgload.
> pkgload::load_all(quiet = TRUE)
ℹ Loading pkgload.
```

I tried to fix this in `load_all_quiet()` but I don't understand the logic there.  Maybe `quiet` needs to be set to `NULL` by default in order for this more complex logic to work?

### PR DESCRIPTION
**Pull request by lionel-**
Closes #213

### CODE REVIEW

File: R/load.R [hunk 1]
```diff
@@ -277,11 +277,17 @@ load_all <- function(path = ".",
...
 load_all_quiet <- function(quiet, fn = NULL) {
   if (!is_null(fn)) {
-    quiet <- peek_option(sprintf("testthat:::%s_quiet_override", fn))
+    # This overwrites the `quiet` setting
+    quiet <- peek_option(sprintf("testthat:::%s_quiet_override", fn)) %||% quiet
   }
-  quiet %||% peek_option("testthat:::load_all_quiet") %||% FALSE
+
+  # This doesn't overwrite the `quiet` setting, only changes the default
+  quiet %||% peek_option("testthat:::load_all_quiet_default") %||% FALSE
```
Comment (hadley) [inline R/load.R:290]: But when is `quiet` ever `NULL` here?

Comment (lionel-) [reply R/load.R:290]: When it is not supplied.

Comment (hadley) [reply R/load.R:290]: Ooh it's `NULL` in `load_all()`, but `FALSE` in `load_code()` and `load_depends()`; that's what was confusing me.

Comment (lionel-) [reply R/load.R:290]: oh hmm, I should probably update these defaults too

**Pull request merged**